plugins/providers/virtualbox/provider.rb

```diff
@@ -56,9 +56,6 @@ def machine_id_changed
 
       # Returns the SSH info for accessing the VirtualBox VM.
       def ssh_info
-        # If the VM is not running that we can't possibly SSH into it
```

Comment:
I think we will need to check if the machine has been booted here. If the machine was never booted, this would return incorrect information.